### PR TITLE
Adding Sentry

### PR DESCRIPTION
**Pull request by RonanDeF**
Added the sentry intializer

### CODE REVIEW

File: profiles/__init__.py [hunk 1]
```diff
@@ -1,10 +1,18 @@
...
+import sentry_sdk
+from sentry_sdk.integrations.flask import FlaskIntegration
+
 import csh_ldap
 from flask import Flask, render_template, jsonify, request, redirect, flash
 from flask_pyoidc.flask_pyoidc import OIDCAuthentication
 from flask_uploads import UploadSet, configure_uploads, IMAGES
 
+sentry_sdk.init(
+    dsn="https://<key>@sentry.io/<project>",
```
Comment (devinmatte) [inline profiles/__init__.py]: This should be pulled from the config

File: profiles/__init__.py [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+    integrations=[FlaskIntegrations()]
```
Comment (devinmatte) [inline profiles/__init__.py]: Add `SqlalchemyIntegrations` also

File: config.env.py [hunk 3]
```diff
@@ -3,6 +3,12 @@
 import string
 from os import environ as env
 
+# Sentry DSN
+SENTRY_DSN = env.get("PROFILES_SENTRY_DSN", "")
+SENTRY_CONFIG = {
```
Comment (devinmatte) [inline config.env.py]: This config is never used and is therefore not needed

File: profiles/__init__.py [hunk 4]
```diff
@@ -17,6 +21,12 @@
 auth = OIDCAuthentication(app, issuer=app.config["OIDC_ISSUER"],
                           client_registration_info=app.config["OIDC_CLIENT_CONFIG"])
 
+# Sentry
+sentry = Sentry(app)
```
Comment (devinmatte) [inline profiles/__init__.py]: Actually looks like it's failing because you don't import Sentry

File: profiles/__init__.py [hunk 5]
```diff
@@ -17,6 +21,12 @@
...
+sentry_sdk.init(
+    dsn=app.config['SENTRY_DSN'],
+    integrations=[FlaskIntegrations(), SqlalchemyIntegration()]
```
Comment (devinmatte) [inline profiles/__init__.py]: Also `FlaskIntegration` not plural

**Comment by devinmatte** [comment]
@RonanDeF Any progress on this?

**Comment by RonanDeF** [comment]
> @RonanDeF Any progress on this?

I got a new laptop that actually works and am currently setting up an env so i can work on this properly. Otherwise not anything major in the sense of progress.

**Review by devinmatte — APPROVED** [review]
We'll get it into dev to test

**Pull request merged**